### PR TITLE
splitting up angular objectives

### PR DESCRIPTION
**Pull request by welzie**
- angular lesson 1
- angular lesson 2
- angular lesson 3 (still at high level state, will need more detail)

### CODE REVIEW

File: unit1/lesson19.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* Structural directives start with a ``*``, such as ``*ngIf``, ``*ngFor``.
+* Only one structural directive can be assigned to a host element.
+* Understand that ``*ngIf`` adds or removes the host element and it's descendents.
+* Know that each iteration of ``*ngFor`` creates a copy of the host element and it's descendents.
+* Can use current item as a local variable
+
+  * Example: ``<li *ngFor="let user of users"> {{user}} </li>``
+  
+* Know how to use local variables ``index``, ``first``, ``last``, ``even``, ``odd``.
+* Handle a click with a ``(click)`` directive and function in component to respond to the click.
```
Comment (jimflores5) [inline unit1/lesson19.rst]: This sentence has unclear phrasing. I can follow the idea up through "...directive and function", but after that I think the sentence loses meaning.

File: unit1/lesson19.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Understand how ``(click)`` relates to DOM ``anElement.addEventListener("click", aFunction);``
+* Use template reference variables and a click handler to take in user input
+
+  * Example: ``<input #name (click)=addThing(name)>``
```
Comment (jimflores5) [inline unit1/lesson19.rst]: Since this is an input box, should the function call be ``addThing(name.value)``?

File: unit1/lesson19.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
+At the end of angular class 2 students will be able to declare click handlers, take in user input and display it, and show lists of data using a loop directive.
+
+Angular Class 2 Objectives:
```
Comment (jimflores5) [inline unit1/lesson19.rst:3]: Nice work on the Angular Class 2 objectives.  These are WAY better than the originals.

File: unit1/lesson19.rst (hunk 1 above)
Comment (welzie) [reply unit1/lesson19.rst]: Adjusted wording lmk if it makes more sense.

File: unit1/lesson19.rst (hunk 2 above)
Comment (welzie) [reply unit1/lesson19.rst]: good catch

**Pull request merged**